tests/integration/security/chiron/dns_cert_test.go

```diff
@@ -48,5 +101,54 @@ func TestDNSCertificate(t *testing.T) {
 			secret.ExamineDNSSecretOrFail(t, galleySecret, galleyDNSName)
 			t.Log(`checking Sidecar Injector DNS certificate is valid`)
 			secret.ExamineDNSSecretOrFail(t, sidecarInjectorSecret, sidecarInjectorDNSName)
+
+			// Test certificate regeneration: if a DNS certificate is deleted, Chiron will regenerate it.
```

Comment:
there are 3 test cases in this function, can we run them separately in `t.Run()` with an explicit name for each test case? This makes it much easier to find out what is failing by just looking at the dashboard without looking further into the log.

Reply:
Done.